prompterator/utils.py

```diff
@@ -362,3 +380,42 @@ def insert_hidden_html_marker(helper_element_id, target_streamlit_element=None):
 def format_traceback_for_markdown(text):
     text = re.sub(r" ", "&nbsp;", text)
     return re.sub(r"\n", "\n\n", text)
+
+
+def build_function_calling_tooling(json_schema):
```

Comment:
Feels like something that should be moved into `models/`, perhaps `models/openai_model_utils.py`?

Reply:
Another thing is that the name of the function is hard to digest, it doesn't say anything about the inputs, outputs, or the process that happens inside to get from the inputs to the outputs. Can you think of a more descriptive name?

Reply:
well, the name is already pretty long so I would prefer adding docstring if you that's acceptable